src/app/code/community/Hackathon/PromoCodeMessages/Model/Validator.php

```diff
@@ -170,6 +170,15 @@ protected function _validateGeneral($rule, $coupon)
             }
         }
 
+        // magemail coupon-level auto-expiration date
+        if ($coupon->getdata('magemail_expired_at') && $coupon->getUsageLimit() && $coupon->getTimesUsed() >= $coupon->getUsageLimit()) {
+            $expirationDate = new Zend_Date($coupon->getdata('magemail_expired_at'), Varien_Date::DATE_INTERNAL_FORMAT);
```

Comment:
@philwinkle Dang son that was a newbie mistake.  Will fix.  I think Zend_Date is being used in a couple places in this class already.